core/src/test/scala/matryoshka/spec.scala

```diff
@@ -175,27 +174,31 @@ object Exp2 {
 }
 
 
-class ExpSpec extends Spec {
+class ExpSpec extends Specification with ScalaCheck with CheckAll {
   import Exp._
 
   implicit val arbExpInt: Arbitrary[Exp[Int]] = arbExp(Arbitrary.arbInt)
   // NB: These are just a sanity check that the data structure created for the
   //     tests is lawful.
-  checkAll(equal.laws[Exp[Int]])
-  checkAll(traverse.laws[Exp])
+  "Exp should satisfy relevant laws" >> {
+    checkAll(equal.laws[Exp[Int]]) append
+    checkAll(traverse.laws[Exp])
+  }
 }
 
-class Exp2Spec extends Spec {
+class Exp2Spec extends Specification with ScalaCheck with CheckAll {
   import Exp2._
 
   implicit val arbExp2Int: Arbitrary[Exp2[Int]] = arbExp2(Arbitrary.arbInt)
   // NB: These are just a sanity check that the data structure created for the
   //     tests is lawful.
-  checkAll(equal.laws[Exp2[Int]])
-  checkAll(functor.laws[Exp2])
+  "Exp2 should satisfy relevant laws" >> {
+    checkAll(equal.laws[Exp2[Int]])
```

Comment:
Yeah, I’m fine with either. But don’t we eventually have to switch to append/and as we move toward the more functional approach to specs2?